package.json

```diff
@@ -0,0 +1,149 @@
+{
+  "name": "terra-application",
+  "main": "lib/application-base/ApplicationBase.js",
+  "version": "1.11.0",
+  "description": "A framework to support application development with Terra components",
+  "engines": {
+    "node": ">=8.9.2 <12"
```

Comment:
```suggestion
    "node": ">=8.9.2 <11"
```

Reply:
Well, actually it built on 11.
![](https://media.giphy.com/media/jPAdK8Nfzzwt2/giphy.gif)